x/txreward/internal/types/codec.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+import "github.com/cosmos/cosmos-sdk/codec"
+
+func RegisterCodec(cdc *codec.Codec) {
+	cdc.RegisterConcrete(MsgIncrementBlockRewardsPool{}, "commercio/incrementBlockRewardsPool", nil)
```

Comment:
I think it's better to have a unified way of naming messages, so we should change this to 

```suggestion
	cdc.RegisterConcrete(MsgIncrementBlockRewardsPool{}, "commercio/MsgIncrementBlockRewardsPool", nil)
```